nipype/info.py

```diff
@@ -141,7 +141,8 @@ def get_nipype_gitversion():
     'configparser',
     'pytest>=%s' % PYTEST_MIN_VERSION,
     'mock',
-    'pydotplus'
+    'pydotplus',
+    'packaging',
```

Comment:
Builds and installs without issue. (Had to apt-get install build-essential, conda install scipy, lxml and dependencies, but otherwise no problems.)